tools/calc-lib-size

```diff
@@ -65,11 +69,21 @@ for arg in "$@"; do
 done
 IFS=' ' read -a args <<< "$args"
 
-CONFIGURE_OPTS="$args"
+if [ $OPT_KEEP_LOG -eq 0 ]; then
+    echo "# logs disabled"
+else
+    echo "# logs enabled"
+fi
+
+echo
 
 function calc_size() {
-    make alldefconfig
+    echo "# building with CFLAGS='$@'"
+    make alldefconfig > /dev/null
     make clean > /dev/null
```

Comment:
usually you clean before you change config, that's to avoid missing files that do not belong in the new configuration.